### PR TITLE
fix: APPS-3424 Fix Footer Spacing

### PR DESCRIPTION
**Pull request by tinuola**
Connected to [APPS-3424](https://jira.library.ucla.edu/browse/APPS-3424)

**Notes:**

- Refactored CSS to standardize spacing above footer across page templates
  - 120px above footer on desktop, 86px on mobile
  - Updated pagination block (top-bottom) spacing to 60px
  - See [Figma specs](https://www.figma.com/design/L8e1czKo8a8px368ayyPZ4/-FTVA--Final-Design-with-Annotations?node-id=10008-3119&t=zuTuUvIzhefD71vn-1)
- Added `page-bottom-spacer` class/selector to global stylesheet
  - Added class to page templates; removed extra/excess spacing across templates
  - Did not apply the class to `search.vue` and `filmmakers/[slug].vue` because it would create a white gap and clash with the pale-blue content element above; so added the spacing rules directly to their templates 
- No changes to Error page and pages that end with `Explore More`  

**Previews:**

- Home: https://deploy-preview-218--test-ftva.netlify.app/
- Search: https://deploy-preview-218--test-ftva.netlify.app/search?q=family
- General Content: https://deploy-preview-218--test-ftva.netlify.app/about
- Billy Wilder: https://deploy-preview-218--test-ftva.netlify.app/billy-wilder-theater/
- ARCS: https://deploy-preview-218--test-ftva.netlify.app/archive-research-study-center
- Blog Listing: https://deploy-preview-218--test-ftva.netlify.app/blog/
- Series Listing: https://deploy-preview-218--test-ftva.netlify.app/series/
- Events Listing: https://deploy-preview-218--test-ftva.netlify.app/events/
- Events Detail: https://deploy-preview-218--test-ftva.netlify.app/events/step-up-high-water-2018-2022-10-18-25
- Collections Index: https://deploy-preview-218--test-ftva.netlify.app/collections/
- Collection Detail: https://deploy-preview-218--test-ftva.netlify.app/collections/robert-abel-associates
- Collection Items Listing: https://deploy-preview-218--test-ftva.netlify.app/collections/la-rebellion/filmography/
- Collection Item Detail: https://deploy-preview-218--test-ftva.netlify.app/collections/la-rebellion/bless-their-little-hearts/
- Collection Type Listing:  https://deploy-preview-218--test-ftva.netlify.app/collections/motion-picture/
- Storytelling: https://deploy-preview-218--test-ftva.netlify.app/collections/la-rebellion/
- Filmmakers Listing: https://deploy-preview-218--test-ftva.netlify.app/collections/la-rebellion/filmmakers/
- Filmmaker Detail-1: https://deploy-preview-218--test-ftva.netlify.app/collections/la-rebellion/filmmakers/gay-abel-bey/
- Filmmaker Detail-2: https://deploy-preview-218--test-ftva.netlify.app/collections/la-rebellion/filmmakers/larry-clark/

**No Change**

- Error: https://deploy-preview-218--test-ftva.netlify.app/error
- Blog Detail: https://deploy-preview-218--test-ftva.netlify.app/blog/the-effervescent-artist-educator-and-activist-betty-chen
- Series Detail: https://deploy-preview-218--test-ftva.netlify.app/series/todd-solondz-series

**Checklist:**

-   [x] I added github label for semantic versioning
-   [x] I double checked it looks like the designs
-   ~~[ ] I completed any required mobile breakpoint styling~~
-   ~~[ ] I completed any required hover state styling~~
-   ~~[ ] I included a working spec file~~
-   ~~[ ] I added notes above about how long it took to build this component~~
-   [x] UX has reviewed this PR
-   [x] I requested a PR review from the Dev team

[APPS-3424]: https://uclalibrary.atlassian.net/browse/APPS-3424?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by farosFreed — CHANGES_REQUESTED** [review]
I didn't review this 100% yet, but I found this and was wondering if the spacing is correct on this page above the footer? It seems larger. Are flexible blocks that end with a horizontal rule an edge case we need to target? Or is there a way to change the styles so they just handle this naturally? https://deploy-preview-218--test-ftva.netlify.app/events/my-octopus-teacher-12-15-24

File: assets/styles/global.scss [hunk 1]
```diff
@@ -24,7 +24,8 @@ body {
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
     font-feature-settings: "kern" 1;
-    font-kerning: normal;
+    -webkit-font-kerning: normal;
```
[the remark this answers is not in the document]
Comment (tinuola) [reply assets/styles/global.scss:27]: The prefixes were not added manually; probably happened when I ran the linter. Should they be removed?

File: pages/[...slug].vue [hunk 2]
```diff
@@ -72,8 +72,8 @@ const parsedCarouselData = computed(() => {
   })
 })
 
-const pageClass = computed(() => {
-  return ['page', 'page-detail', 'page-detail--paleblue', 'page-general-content', path]
+const pageClasses = computed(() => {
+  return ['page', 'page-detail', 'page-detail--paleblue', 'page-general-content', 'page-bottom-spacer']
```
[the remark this answers is not in the document]
Comment (tinuola) [reply pages/[...slug].vue:76]: @farosFreed An accidental removal. I'll add it back. Thanks for the catch.

**Review by farosFreed — APPROVED** [review]
Great work
